src/components/compose-input.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+// If you uncomment this, it will fail to build, because Next isn't a safe import?
+// Some bloom-filter thing is failing to resolve buffer
+// import { useRouter } from "next/router";
```

Comment:
Uncomment this + the hook call and this is the result

<img width="1802" alt="image" src="https://user-images.githubusercontent.com/6751787/223313507-999a2a20-8ffd-4e9e-a3a8-551f473efde2.png">